mep/books/admin.py

```diff
@@ -62,6 +65,25 @@ def borrow_count(self, obj):
     # use the annotated queryset value to make the field sortable
     borrow_count.admin_order_field = 'borrow__count'
 
+    # add custom fields to the tabular export
+    def tabular_headers(self, queryset):
+        return get_field_names_from_queryset(queryset) + ['authors', 'url']
+
+    def tabulate_queryset(self, queryset):
+        fields = get_field_names_from_queryset(queryset)
+        for item in queryset.prefetch_related('creator_set'):
+            values = [getattr(item, field) for field in fields]
+            # logic for custom tabular export fields
+            values.extend((';'.join([str(auth) for auth in item.authors]),
```

Comment:
Can we use the existing `author_list` property on `Item`? Maybe replace `,` with `;` here before including?

Reply:
I think if we do a simple `replace()` we'll actually break the data, because the authors in `author_list` are added by sort name, which means most of them have a comma already there. For multiple authors that would result in something like: `Bronte; Charlotte; Hemingway; Ernest`.

Reply:
Maybe we could just change `author_list` itself to use a `;`? Looking at the item list, I see stuff like: `Beaumont, Francis, Fletcher, John` which isn't very clear to me...although maybe that's the standard?

Reply:
Eww, we're generating a comma-separated list of author names with commas in them? Yes, lets fix that.